Samples/WinMLSamplesGallery/WinMLSamplesGallery/Common/Sample.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections.Generic;
+
+namespace WinMLSamplesGallery
+{
+    public sealed class Sample
```

Comment:
Can you name this class something more descriptive? "Sample" is pretty generic, it would be good to be a little more detailed on what this class is for.

Reply:
Changed to SampleMetadata and added a description